### DISCUSSION TITLE
Licence

### DISCUSSION THREAD

**Discussion by sn0b4ll** [austinsonger/Incident-Playbook#111] [General]
Hey Austin,
First of hughe props, I really like the repo and the structure and would like to use and contribute to it. Sadly there is currently no licence assigned, which means, that you retain full rights on the things which are in this repo. This hinders me from collaborating in my work-time, since we are only allowed to use / contribute to project with licences which allow collaboration.

Would you mind choosing a licence (if you don't care maybe something like MIT or BSD-3-Clause)?

Thank you in advace and again thanks for the great work so far 👍 
sn0b4ll

**Comment by austinsonger** [comment]
Done.

**Reply by sn0b4ll** [reply]
Perfect, thank you!
